internal/tutils/psql.go

```diff
@@ -49,6 +49,8 @@ func NewPSQLContainer() (*PSQLContainer, error) {
 	out, err := exec.Command(
 		rt,
 		"run",
+		"--mount=type=tmpfs,destination=/var/lib/postgresql/data",
+		"--mount=type=tmpfs,destination=/dev/shm",
```

Comment:
This significantly speeds up on one of my systems (old Xeon with HDD). From 336s to just 36s.